src/cmd/test.ts

```diff
@@ -0,0 +1,55 @@
+import { unlinkSync, writeFileSync } from 'fs'
```

Comment:
I notice Dockerfile still uses `bin/ci-tests.sh` for the test phase, should I expect that will continue or will we port it as well?